Tests/TSCUtilityTests/NetrcTests.swift

```diff
@@ -0,0 +1,449 @@
+import XCTest
+import TSCUtility
+
+@available(macOS 10.13, *)
+class NetrcTests: XCTestCase {
+    func testLoadMachinesInline() {
+        //			it("should load machines for a given inline format") {
```

Comment:
**nitpick:** I don't think these type of comments are used anywhere else in the codebase. If there's a need to explain what a test case does, I think the usual `///` doc comments would be more suitable.

Reply:
Also, this is applicable to the rest of the instances of `// it("` in this PR